src/models/azure-identity/test-data.ts

```diff
@@ -12,23 +12,27 @@ export const testData: TestDependencyDataType<AzureIdentityModel> = [
     __testDescription: 'Valid full object',
     clientId: 'any-client-id',
     serviceAccountName: 'any-sa',
+    azureKeyVaults: ['kv1', 'kv2'],
   },
```

Comment:
add valid test for partial object
```suggestion
  },
  {
    __testDescription: 'Valid partial object',
    clientId: 'any-client-id',
    serviceAccountName: 'any-sa',
  },
```